From: Agustin Henze <tin@sluc.org.ar>
Date: Mon, 18 Nov 2013 19:05:59 -0300
Subject: Making bootstrap as default theme, not bootstrap3

---
 nikola/plugins/command/init.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nikola/plugins/command/init.py b/nikola/plugins/command/init.py
index 1873ec4..59a27e0 100644
--- a/nikola/plugins/command/init.py
+++ b/nikola/plugins/command/init.py
@@ -64,7 +64,7 @@ class CommandInit(Command):
         'BLOG_EMAIL': "joe@demo.site",
         'BLOG_DESCRIPTION': "This is a demo site for Nikola.",
         'DEFAULT_LANG': "en",
-        'THEME': 'bootstrap3',
+        'THEME': 'bootstrap',
 
         'POSTS': """(
     ("posts/*.rst", "posts", "post.tmpl"),
